app/models/concerns/hmis_structure/client.rb

```diff
@@ -7,13 +7,20 @@
 module HmisStructure::Client
   extend ActiveSupport::Concern
   include ::HmisStructure::Base
+  include HasPiiAttributes
 
   included do
     # pg generated columns for search
     self.ignored_columns += [:search_name_full, :search_name_last]
     self.hud_key = :PersonalID
     self.additional_upsert_columns = [:demographic_dirty]
     acts_as_paranoid(column: :DateDeleted) unless included_modules.include?(Paranoia)
+
+    pii_attr :SSN
+    pii_attr :FirstName
+    pii_attr :MiddleName
+    pii_attr :LastName
+    pii_attr :DOB
```

Comment:
@ttoomey we've aliased these to underscore format like `:first_name` which can be used in most contexts, do you know if those would be a problem here?  I'm leaning toward what you did (use the same shape as the database column name) but it might be nice to use the aliased versions if that would work.

Reply:
it should be fine to convert these to snake case, either should work. I think it's probably better to use snake case generally, I can change that